### ISSUE TITLE
Not able to install 

### ISSUE THREAD

**Issue by DecentMakeover** [bukalapak/pybrisque#3]
Hi 

Thanks for sharing your work.

but when i type `pip install --process-dependency-links pybrisque` , i get 

`no such option: --process-dependency-links`

Any suggestions would be helpful.

Thanks in advance.

**Comment by madcat1991** [comment]
`--process-dependency-links` has been removed in pip>19. So a strange, but working for me option is  to downgrade your `pip` to version 18.1 and repeat the installation.

```
$ pip install -U pip==18.1
Collecting pip==18.1
  Using cached https://files.pythonhosted.org/packages/c2/d7/90f34cb0d83a6c5631cf71dfe64cc1054598c843a92b400e55675cc2ac37/pip-18.1-py2.py3-none-any.whl
Installing collected packages: pip
  Found existing installation: pip 19.0.2
    Uninstalling pip-19.0.2:
      Successfully uninstalled pip-19.0.2
Successfully installed pip-18.1
```

```
$ pip install --process-dependency-links pybrisque
Collecting pybrisque
  Using cached https://files.pythonhosted.org/packages/0b/38/2bafdc76e506df9e6ea6ca636e1fe291f2b40119c1c347f4a240d8ee3300/pybrisque-1.0.tar.gz
DEPRECATION: Dependency Links processing has been deprecated and will be removed in a future release. A possible replacement is PEP 508 URL dependencies. You can find discussion regarding this at https://github.com/pypa/pip/issues/4187.
Requirement already satisfied: numpy in /Users/tural/.virtualenvs/CV/lib/python3.7/site-packages (from pybrisque) (1.15.2)
Requirement already satisfied: scipy in /Users/tural/.virtualenvs/CV/lib/python3.7/site-packages (from pybrisque) (1.1.0)
Requirement already satisfied: opencv-python in /Users/tural/.virtualenvs/CV/lib/python3.7/site-packages (from pybrisque) (4.0.0.21)
Collecting libsvm (from pybrisque)
  Cloning https://github.com/akbargumbira/libsvm-python.git (to revision master) to /private/var/folders/_p/qr72_vb92jlg29h36dl397ww0000gn/T/pip-install-47bo8627/libsvm
DEPRECATION: Dependency Links processing has been deprecated and will be removed in a future release. A possible replacement is PEP 508 URL dependencies. You can find discussion regarding this at https://github.com/pypa/pip/issues/4187.
Building wheels for collected packages: pybrisque, libsvm
  Running setup.py bdist_wheel for pybrisque ... done
  Stored in directory: /Users/tural/Library/Caches/pip/wheels/4a/f2/58/326df6416180141128635d5a2d0f9678655da3f47cbe975285
  Running setup.py bdist_wheel for libsvm ... done
  Stored in directory: /private/var/folders/_p/qr72_vb92jlg29h36dl397ww0000gn/T/pip-ephem-wheel-cache-iau8_24h/wheels/b2/b5/86/16d5fb4760bc76e7a224ef24e64f7c2e1b66c17de94aa4aa6f
Successfully built pybrisque libsvm
Installing collected packages: libsvm, pybrisque
Successfully installed libsvm-3.14 pybrisque-1.0
You are using pip version 18.1, however version 19.0.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

**Comment by chungyau97** [comment]
> `--process-dependency-links` has been removed in pip>19. So a strange, but working for me option is to downgrade your `pip` to version 18.1 and repeat the installation.
> 
> ```
> $ pip install -U pip==18.1
> Collecting pip==18.1
>   Using cached https://files.pythonhosted.org/packages/c2/d7/90f34cb0d83a6c5631cf71dfe64cc1054598c843a92b400e55675cc2ac37/pip-18.1-py2.py3-none-any.whl
> Installing collected packages: pip
>   Found existing installation: pip 19.0.2
>     Uninstalling pip-19.0.2:
>       Successfully uninstalled pip-19.0.2
> Successfully installed pip-18.1
> ```
> ```
> $ pip install --process-dependency-links pybrisque
> Collecting pybrisque
>   Using cached https://files.pythonhosted.org/packages/0b/38/2bafdc76e506df9e6ea6ca636e1fe291f2b40119c1c347f4a240d8ee3300/pybrisque-1.0.tar.gz
> DEPRECATION: Dependency Links processing has been deprecated and will be removed in a future release. A possible replacement is PEP 508 URL dependencies. You can find discussion regarding this at https://github.com/pypa/pip/issues/4187.
> Requirement already satisfied: numpy in /Users/tural/.virtualenvs/CV/lib/python3.7/site-packages (from pybrisque) (1.15.2)
> Requirement already satisfied: scipy in /Users/tural/.virtualenvs/CV/lib/python3.7/site-packages (from pybrisque) (1.1.0)
> Requirement already satisfied: opencv-python in /Users/tural/.virtualenvs/CV/lib/python3.7/site-packages (from pybrisque) (4.0.0.21)
> Collecting libsvm (from pybrisque)
>   Cloning https://github.com/akbargumbira/libsvm-python.git (to revision master) to /private/var/folders/_p/qr72_vb92jlg29h36dl397ww0000gn/T/pip-install-47bo8627/libsvm
> DEPRECATION: Dependency Links processing has been deprecated and will be removed in a future release. A possible replacement is PEP 508 URL dependencies. You can find discussion regarding this at https://github.com/pypa/pip/issues/4187.
> Building wheels for collected packages: pybrisque, libsvm
>   Running setup.py bdist_wheel for pybrisque ... done
>   Stored in directory: /Users/tural/Library/Caches/pip/wheels/4a/f2/58/326df6416180141128635d5a2d0f9678655da3f47cbe975285
>   Running setup.py bdist_wheel for libsvm ... done
>   Stored in directory: /private/var/folders/_p/qr72_vb92jlg29h36dl397ww0000gn/T/pip-ephem-wheel-cache-iau8_24h/wheels/b2/b5/86/16d5fb4760bc76e7a224ef24e64f7c2e1b66c17de94aa4aa6f
> Successfully built pybrisque libsvm
> Installing collected packages: libsvm, pybrisque
> Successfully installed libsvm-3.14 pybrisque-1.0
> You are using pip version 18.1, however version 19.0.3 is available.
> You should consider upgrading via the 'pip install --upgrade pip' command.
> ```

Thank You

**Comment by akbargumbira** [comment]
Yes, apparently --process-dependency-link has been obsolete. We'll update the installation. For now, downgrading pip can be a temporary solution..

**Comment by Abhishek9900** [comment]
To install it with latest pip>=19 first install libsvm separately with `apt-get install libsvm-dev` then install the module with `pip3 install pybrisque --no-deps` . Then you'll have to install libsvm for python with this command `pip3 install -e git+https://github.com/Salinger/libsvm-python.git#egg=libsvm-python` .

**Comment by akbargumbira** [comment]
See https://github.com/cjlin1/libsvm/pull/145 for a solution that would fix this problem

**Comment by chiradeepdey** [comment]
Instead of downgrading  pip, doing: 
`pip install pybrisque `
seemed to work fine for me

**Comment by kkedich** [comment]
Executing `pip install pybrisque` did not worked for me. But the solution of @Abhishek9900 did, with a little change in the egg name: `libsvm-python` was not available but `libsvm` was. The environment that I was testing included python=3.8 and pip=21.0.1

1. `apt-get install libsvm-dev`
2. `pip3 install pybrisque --no-deps`
3. `pip3 install -e git+https://github.com/Salinger/libsvm-python.git#egg=libsvm`

**Comment by yaowangvrviu** [comment]
change REQUIREMENTS.txt.
-e git+https://github.com/Salinger/libsvm-python.git#egg=libsvm-python
to
-e git+https://github.com/Salinger/libsvm-python.git#egg=libsvm
